packages/discussions/graphql/resolvers/Discussion/comments.js

```diff
@@ -305,6 +305,10 @@ module.exports = async (discussion, args, context, info) => {
         })
     }
     if (first) {
+      if (maxDepth != null) {
```

Comment:
This breaks expanding the comment tree in the frontend. If a `parentId` is provided the frontend expects to get `flatDepth` from the parent id. Going to revert now, so it works again in prod. Let's figure out the solution later.

Reply:
commented out with the following commit: https://github.com/orbiting/backends/commit/4223a6298368db404bb2940072d19a9d88be06ef